api/tx_args.go

```diff
@@ -154,33 +157,43 @@ func (args *SendTxArgs) setDefaults(ctx context.Context, b Backend) error {
 	}
 	// For the transaction that do not use the gasPrice field, the default value of gasPrice is not set.
 	if args.Price == nil && *args.TypeInt != types.TxTypeEthereumDynamicFee {
-		// TODO-Klaytn: needs to be fixed to double the BaseFee
+		// b.SuggestPrice = unitPrice, for before KIP71
+		//                = baseFee,   for after KIP71
 		price, err := b.SuggestPrice(ctx)
```

Comment:
Don't we need to set `baseFee * 2` after KIP-71 hard fork?

Reply:
As in doCall in the direction of consistency, I think It would be better to set `baseFee * 2`

Reply:
Ohh @jimni1222, we multiply `baseFee` by 2 in L185.

Reply:
This logic is not for `TxTypeEthereumDynamicFee`, so we need to set `args.Price` with `baseFee * 2` in my opinion.

Reply:
Right, It's good to keep the consistency.